python/src/wslink/websocket.py

```diff
@@ -172,6 +172,12 @@ def setServerProtocol(self, newServerProtocol):
     def getServerProtocol(self):
         return self._protocolHandler
 
+    ### Does not seem to work to print a "ready line"
+    # def startFactory(self):
```

Comment:
Debugging, Twisted seems to call startFactory on a twisted.web.server.Site object, so http.py HttpServer::startFactory() gets called. We might be able to provide an override for that in wslink.